queue/QueueRunner.feature

```diff
@@ -5,8 +5,8 @@ Feature: Command and control using a message broker
 
   Scenario: Default client setting
     Then the time to wait for requests is 500ms
-    Then the request queue is "testuser.req"
-    Then the response queue is "testuser.resp"
+    Then the request queue is "testuser-req"
+    Then the response queue is "testuser-resp"
```

Comment:
This is a tricky change. We need to make sure that the client is backwards compatible.
The queue names are driven by the server, so we should not change them here.

We might need to support two different queue naming convention, one for JMS and one for SQS

Reply:
As far as the test is concerned, it should read it off some parameter passed in via some environment variabe or local config file or something of that sort, and not have it hard-code

Reply:
Also on another note, is it really needed to do this test? Because if this test fails all other tests would fail,  the key is to have the correction config but to check for it again might be redundant.

Reply:
Hey Mani, 

I like the idea of injecting the queue name via constructor parameters.
Bare in mind that we are developing a library (tdl-client-java) and not the application (tdl-runner-java). 

The queue names can be added to the credentials.config file upon generation.

Would you be able to change the Spec so that it reflects this?

Reply:
While I'm at it I'm going to give that a try, via a config, any pattern you would like to follow? Given there are config files around already? I could reuse one of them and then makes it easier to use the same API to read the config and check for it

Reply:
After this, the Spec should be good to go.

Reply:
Where is the `credentials.config` stored, I can't find it on tdl-client, not client to me which config file you are referring to

Reply:
The credentials.config is specific to the `tdl-runner-*`, you should not worry about it at this time. The client can be constructed with the queue names.

Reply:
Also how would you like the spec to be changed, we still have the same issue checking for specific names of the queue:
```cucumber
  Scenario: Default client setting
    Then the time to wait for requests is 500ms
    Then the request queue is "testuser.req"   <=== these 
    Then the response queue is "testuser.resp" <== names cannot be checked for via such a test
```

Reply:
I have removed a part of the test, its brittle and does not deliver as much value.